app/routes/events/components/RegistrationMeta.js

```diff
@@ -70,23 +71,23 @@ const PresenceStatus = ({
   switch (presence) {
     case 'NOT_PRESENT':
       return (
-        <>
+        <div>
           <i className="fa fa-exclamation-circle" /> Du møtte ikke opp på
           arrangementet
-        </>
+        </div>
       );
     case 'PRESENT':
       return (
-        <>
+        <div>
           <i className="fa fa-check-circle" /> Du møtte opp på arrangementet
```

Comment:
I was considering whether "Du deltok på arrangementet" is better (especially since "Du er påmeldt" is removed once event has ended)? Any thoughts?

Reply:
I like it